### PR TITLE
PR for simplified grid.

### PR DESCRIPTION
**Pull request by adidoshi**
### Hey I create simplified grid component, please review.
* grid-2: two items in a grid
* grid-3: three items

### CODE REVIEW

File: components/simplifiedGrid/SimplifiedGrid.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+.grid-col-2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px;
+}
+
+.grid-col-3 {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (msdeshmukh009) [inline components/simplifiedGrid/SimplifiedGrid.css:12]: You can use `repeat(3,1fr)` if you want.

File: components/simplifiedGrid/SimplifiedGrid.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  gap: 10px;
+}
+
+.grid-item {
+  box-shadow: 0 5px 20px -5px rgb(191 192 192);
```
Comment (msdeshmukh009) [inline components/simplifiedGrid/SimplifiedGrid.css:17]: A variable can be used here as well.

**Review by msdeshmukh009 — APPROVED** [review]
The Ui is looking good. And components are working well. Nice work!!. Also, you can add media queries so the components will wrap rather than shrink.

**Review by srishti-maurya — APPROVED** [review]
looks good

File: components/simplifiedGrid/SimplifiedGrid.css (hunk 1 above)
Comment (adidoshi) [reply components/simplifiedGrid/SimplifiedGrid.css:12]: Thanks, making the changes.

File: components/simplifiedGrid/SimplifiedGrid.css (hunk 2 above)
Comment (adidoshi) [reply components/simplifiedGrid/SimplifiedGrid.css:17]: Yes, adding global variable for box shadow, thanks.

**Pull request merged**